.github/workflows/pre-submit.markdown.yml

```diff
@@ -0,0 +1,27 @@
+name: markdown
+
+on:
+  push:
+    branches: [main, "*"]
+  pull_request:
+    # The branches below must be a subset of the branches above
+    branches: [main]
+  workflow_dispatch:
+
+permissions:
+  contents: read # Needed to check out the repo.
+
+jobs:
+  markdow-toc:
+    name: markdown-toc
+    runs-on: ubuntu-latest
+    steps:
+      - name: Setup Node.js 16
+        uses: actions/setup-node@8c91899e586c5b171469028077307d293428b516 # tag=v3.5.1
+        with:
+          node-version: 16
+      - name: Install markdown-toc
+        run: npm install -g markdown-toc
```

Comment:
note: scorecard will probably complain about this dependency not being pinned.

Reply:
Yeah, I have to do the same thing for the pip install of yamllint. I'll fix it at some point.

Reply:
https://github.com/slsa-framework/slsa-github-generator/issues/1661